src/index.html

```diff
@@ -1,13 +1,277 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MYBIKE</title>
+    <link rel="icon" href="images/icon/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__nav">
+        <a
+          id="#home"
+          href="#"
+          class="header__logo"
+          alt="MyBiKE"
+        >
+        </a>
+        <div class="header__elements">
+          <a href="tel:+1 234 555-55-55" class="header__call"></a>
+          <a class="header__menu-opener" href="#menu"></a>
+        </div>
+      </div>
+      <h1 class="header__title">
+          Take the Streets
+      </h1>
+      <nav class="page__menu menu" id="menu">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a
+              href="#"
+              class="menu__logo"
+              alt="MyBiKE"
+            >
+            </a>
+            <a class="menu__cross" href="#"></a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#aboutUs" class="menu__link">About Us</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#compareBikes" class="menu__link">Compare Bikes</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#details" class="menu__link">Details</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contacts</a>
+            </li>
+            <a
+              href="tel:+1 234 555-55-55"
+              class="menu__call menu__call--phone"
+            >
+              +1 234 5555-55-55
+            </a>
+            <a
+              href="tel:+1 234 555-55-55"
+              class="menu__drive menu__drive--action"
+            >
+              Book a test ride
+            </a>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="page__section move">
+        <div class="container-1">
```

Comment:
Why do we need all these different containers? "container-1", "container-2", etc.?
In order to set margins, we have a `page` block, with the help of which we position elements in it